src/routes/(app)/admin[[hash=admin_hash]]/challenge/new/+page.server.ts

```diff
@@ -42,7 +42,7 @@ export const actions: Actions = {
 				),
 				goalAmount: data.get('goalAmount'),
 				mode: data.get('mode'),
-				currency: data.get('currency'),
+				currency: data.get('currency') ?? data.append('currency', 'BTC'),
```

Comment:
```suggestion
				currency: data.get('currency') ?? undefined,
```

https://developer.mozilla.org/fr/docs/Web/API/FormData/append

It returns `undefined`...